### ISSUE TITLE
Unity webgl support

### ISSUE THREAD

**Issue by PudyJapan** [playgameservices/play-games-plugin-for-unity#590]
Hi

I have used this unity plugin on my android application and it works like a charm. :+1: 
Now I need to port my game to webgl and want users to have the same social experience of the google play services.
Does this plugin support the webgl build?

Any help is greatly appreciated.

**Comment by nosmirck** [comment]
+1 to this! please, anyone can give some instructions?

**Comment by claywilkinson** [comment]
Closing as a dupe of #251 - We can track progress there.  Does anyone know if any Unity stats of how many webgl games are being published?

**Comment by bricevdm** [comment]
Unity is moving away from the web plugin (chrome dropped support for NPAPI plugins in September 2015). So, ultimately, expect every browser-based unity game to use WebGL over the next few months/years. 

tldr; relevance: MUCH

**Comment by Disturbing** [comment]
Yes please would like to see this!
